### ISSUE TITLE
(edit: solved, please ignore) Date field problem

### ISSUE THREAD

**Issue by rsacher** [volosoft/jtable#475]
THIS BUG is solved in the current release.
Sorry, I did not use the current release when posting this bug!

---

original message:

I have a date field, which can be left empty.
If it is left empty on creating/updating a record, it will SHOW TODAYs DATE in the table view !
At this stage it has not been entered in the database.
BUT if the user happens to update that record once more, it shows in the edit-form - and then it will be written to the database!

In other words:
JTable will show a date (=today) if a date-field is left empty on create/update !

How can I avoid this?
The list should not show a date, if no date was entered in create/update-forms!

Thanks,
Reinhard
